src/rat/core/run_routing.py

```diff
@@ -302,7 +305,7 @@ def gathering_ro_ou(wkspc_dir, gathered_ro_ou_dir):
     assert wkspc_dir.exists()
     gathered_ro_ou_dir.mkdir(parents=True, exist_ok=True)
     # for all files with this pattern in wkspc dir
-    for f in list(wkspc_dir.glob('**/ou/*.day')):
+    for f in list(wkspc_dir.glob('**/*ou/*.day')):
```

Comment:
wkspc_dir should not be different for forecast and non_forecast mode? Otherwise 'forecast_ou/*.day' will also get copied in non-forecast mode as they will also be there in wkspc_dir.

Reply:
Thanks for the catch. Added an if-else statement to use different glob patterns for forecast/non-forecast modes.